railway/scripts/test_notification.py

```
@@ -3,47 +3,24 @@
 Slack通知テストスクリプト
 """
 
-import os
 import sys
+from datetime import datetime
 
 # プロジェクトルートをパスに追加
 sys.path.append('.')
 
-from railway import SlackNotifier, validate_configs
+from railway.config.settings import validate_configs
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)